aiida/cmdline/commands/cmd_profile.py

```diff
@@ -92,22 +95,36 @@ def profile_setdefault(profile):
     help='Include deletion of associated file repository.'
 )
 @arguments.PROFILES(required=True)
-def profile_delete(force, include_config, include_db, include_repository, profiles):
-    """
-    Delete one or more profiles.
+@click.pass_context
+def profile_delete(ctx, force, include_config, include_database, include_repository, profiles):
+    """Delete one or more profiles.
 
-    You can specify more profile names (separated by spaces).
-    These will be removed from the aiida config file,
-    and the associated databases and file repositories will also be removed.
+    The PROFILES argument takes one or multiple profile names that will be deleted. Deletion here means that the profile
+    will be removed including its file repository and database and all their content. The various options can be used
```

Comment:
(minor) I would say that removing the file repository and the database already makes it sufficiently clear that this includes their content. not important though